### ISSUE TITLE
Outside Corner Relief

### ISSUE THREAD

**Issue by Nanclus** [marvingreenberg/FaceSVG#12]
Thank you SOOOO much for this plug-in, this is amazing!!!

You video you made of a box joint is a perfect example... but I was trying to apply the corner relief to a non-closed face like you would have to do in a box joint. I tried to use the plug in to do that but I couldn't get it to work? Is that supported? If it is, do you have some instructions for me sorry.

I did get the "closed" corner relief's to work thanks so much!!
Jordan

**Comment by marvingreenberg** [comment]
Yeah, it occurred to me that corner relief in that case is probably desirable.  You can just use Sketchup to edit the SVG face group - make a small arc of the right size and then copy it to all the corners manually.  But the software doesn't support it. 

I posted how you might do this as using Sketchup with the existing plugin - it's a kludge and took me a few tries to make it work.  Check out https://www.youtube.com/watch?v=Y0xzSse7MAs 

I'll leave the case open - maybe I'll do something with it some weekend.
